_source/_posts/2022-01-27-build-website-with-eleventy.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: blog_post
+title: "How to Build a Website With Eleventy"
+author: nickolas-fisher
+by: contractor
+communities: [javascript]
+description: "A tutorial that shows you how to build a website with Eleventy and Okta."
+tags: [eleventy, sign-in-widget, markdown, javascript]
+tweets:
+- "Curious about @eleven_ty? Learn how to create a secure Eleventy app using @okta! 🤯"
+image: blog/build-website-with-eleventy/website-eleventy-social.jpg
+type: conversion
+github: https://github.com/oktadev/okta-website-eleventy-example
+---
+
+11ty is a fantastic tool for quickly creating static sites using a variety of templating languages. 11ty is designed to make designing and developing static sites simple. It supports HTML, Markdown, JavaScript, Nunjucks, Handlebars, and many other template styles. It has built-in support for layouts, pagination, and slugify. 11ty aims to compete against other frameworks such as Jekyll, Hugo, Hexo, Gatsby, and Nuxt. The framework is indeed as simple and powerful as they claim to be. 
```

Comment:
```suggestion
11ty is a fantastic tool for quickly creating static sites using a variety of templating languages. 11ty makes designing and developing static sites simple. It supports HTML, Markdown, JavaScript, Nunjucks, Handlebars, and many other template styles, along with built-in support for layouts, pagination, and slugify. 11ty aims to compete against other frameworks such as Jekyll, Hugo, Hexo, Gatsby, and Nuxt. The framework is indeed as simple and powerful as its creators claim. 
```
Avoids repetitious sentence openings, reduces passive voice, avoids ambiguity of closing sentence.